platform/linux-generic/test/pktio_ipc/ipc_common.c

```diff
@@ -159,7 +165,8 @@ void usage(char *progname)
 	       "  -n, --ns           IPC name space ID /dev/shm/odp-<ns>-objname.\n"
```

Comment:
The help is still a bit outdated and could be fixed, `-n` option does not seem to be parsed, so could be dropped (also from help "E.g.") and the `pid` is an option but not documented.

Reply:
Added a new patch to fix this in v3.